src/components/profile/utils.js

```diff
@@ -42,6 +43,58 @@ export async function handleLoanMutationUpdates(
   }
 }
 
+/**
+ * Get an url for the profile page - loans, reservation, orderhistory or cart
+ * @param materialId
+ * @param materialType
+ * @returns {"/work/?type="|string}
+ */
+export function getWorkUrlForProfile({
+  workId = "",
+  pid = "",
+  materialId = "",
+  materialType = "",
+}) {
+  // workid is given
+  if (!isEmpty(workId)) {
+    return `/work/${workId}${
+      materialType ? "?type=" + materialType.toLowerCase() : ""
+    }`;
+  }
+  if (!isEmpty(pid) && isPid(pid)) {
+    return `/linkme.php?rec.id=${pid}`;
+  }
+  // @TODO - materialId may be a localid - that is NOT a faust number
+  // we check if given id is 8 digits - as a faust always is .. but .. is that good enough ..
+  // a localid might also be 8 digits - and not be a faust
+  if (materialId.length === 8) {
+    // we assume that this is a faust
+    return `/linkme.php?faust=${materialId}`;
```

Comment:
Vi kan lave en ny redirect `/work/faust/${faust}`, hvis du har lyst

Reply:
det er jeg ligeglad med - det er vist bare to forskellige måder at gøre det samme